app/src/main/java/com/github/versus/LocationFragment.java

```diff
@@ -108,12 +115,11 @@ public class LocationFragment extends Fragment implements OnMapReadyCallback {
 
     private static ListView listView ;
     private static boolean hasLocations = false;
-    private static Circle mapCircle;
-    private List<CustomPlace> customPlaces = Arrays.asList(new CustomPlace("UNIL Football", "UNIL Football", new LatLng(46.519385, 6.580856)),
-            new CustomPlace("Chavannes Football", "Chavannes Football", new LatLng(46.52527373363714, 6.57366257779824)),
-            new CustomPlace("Bassenges Football", "Bassenges Football", new LatLng(46.52309381914529, 6.5608807098372175)),
-            new CustomPlace("GooglePlex Football", "Google Football", new LatLng(37.422083, -122.082555)));
+    private Marker lastClickedMarker;
+    private Circle lastDrawnCircle;
+    private DummyLocationManager dummyLocationManager;

```

Comment:
Isn't it better to have an higher abstraction of this type (an interface perhaps) ?

Reply:
Yes , I agree ! I've changed it in my last commit !